src/App.tsx

```diff
@@ -17,35 +25,47 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={
+              ({ isActive }) => classNames('navbar-item',
```

Comment:
To improve the readability of the code, it is better to transfer the definition of classes to separate variables and use the ready-made result. You can see an example of use on the website of the [classnames ](https://www.npmjs.com/package/classnames)package.
This applies to all code.